src/util/tracking.ts

```diff
@@ -61,8 +59,13 @@ export type TrackingEventName =
   | 'Survey_Discover2'
   | 'purchase'
   | 'Visa_Card_Launch'
-  | 'Earn_Spend_Launch' // No longer used
   | LoginTrackingEventName
+  /** @deprecated No longer used because it violates our privacy policy */
+  | 'Buy_Quote_Change_Provider'
```

Comment:
Just remove unused ones completely. You can leave comments to note the ones no longer active if you want.